badge.html

```diff
@@ -0,0 +1,164 @@
+                </li>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">Alert</a>
+                </li>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">Alert</a>
+                </li>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">Alert</a>
+                </li>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">Alert</a>
+                </li>
+                
+                
+            
+            </ui>
+        </section>
+
+        <section class="component-doc">
+            <h1 class="component-doc-no">02</h1>
+            <h1 class="component-navbar-heading">badge</h1>
+            <h3 class="component-navbar-item">Badges are used to highlight an item's status for quick recognition.</h3>
+
+            <!-- badge  -->
+            <div class="component-avatar flex flex-nowrap flex-centre">
+                <span class="badge badge-success">success</span>
+                <span class="badge badge-error">error</span>
+                <span class="badge badge-warning">warning</span>
+                <span class="badge badge-info">info</span>
+            </div>
+
+            <!-- /* badge on avatar online, ofline, ddnd */ -->
+            <div class="component-avatar flex flex-nowrap flex-centre">
+                <div class="badge-on-avatar">
+                    <img src="images/60111.jpg" alt="" class="avatar avatar-s"> 
+                    <span class="badge-span badge-online"></span>
+                </div>
+                <div class="badge-on-avatar">
+                    <img src="images/60111.jpg" alt="" class="avatar avatar-md"> 
+                    <span class="badge-span badge-offline"></span>
+                </div>
+                <div class="badge-on-avatar">
+                    <img src="images/60111.jpg" alt="" class="avatar avatar-l">
+                    <span class="badge-span badge-dnd"></span>
+                </div>
+               
+            </div>
+
+            <!-- badge on icons -->
+            <div class="component-avatar flex flex-nowrap flex-centre">
+                <div class="badge-on-avatar">
+                    <i class="fas fa-laugh-wink fa-2x color"></i>
+                    <span class="badge-count">2</span>
+                </div>
+                <div class="badge-on-avatar">
+                    <i class="fas fa-user-secret fa-2x color"></i>
+                    <span class="badge-count">2-</span>
+                </div>
+                <div class="badge-on-avatar">
+                    <i class="fas fa-bug fa-2x color"></i>
+                    <span class="badge-count">2+</span>
+                </div>
+               
+            </div>
```

Comment:
The Icons look good, they can be a little spaced out to use the leftover space in their container.

Reply:
done!!